reactapp/src/redux/toAPI.js

```diff
@@ -0,0 +1,18 @@
+export function toAPI(store, action, conf) {
+    const state = store.getState();
+    fetch('http://' + state.process.API.host + ':' + state.process.API.port + '/api' + conf.url, {
```

Comment:
NO need to store API host, etc in redux store. Just keep that in .env file
and get like ```process.env.REACT_APP_API_HOST``` or smth like that